Tasks/src/StringArrayFilter.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Scanner;
+
+public class StringArrayFilter {
+
+    public static void main(String[] args) {
+
+        final String Result = "Результат вашего запроса следующий :";
```

Comment:
uppercase it's RESULT and so on. Please rewrite

Reply:
Changed all letters in constants to uppercase